Clicker/ViewControllers/EditQuestionTypeViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  EditQuestionTypeViewController.swift
+//  Clicker
+//
+//  Created by Matthew Coufal on 9/18/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol EditQuestionTypeDelegate {
+    func didPick(questionType: QuestionType)
```

Comment:
Can you rename this function to be `editQuestionTypeViewControllerDidPick(...)`. This helps us to easily tell which classes these delegate methods belong to